src/util/pixelSetter.js

```diff
@@ -1,3 +1,4 @@
+/** Sets the pixels from 0 through length of value. */
```

Comment:
Is it possible to document these parameters `x`,`y`,`value` etc using JSDoc?
https://jsdoc.app

Reply:
@HarshKhandeparkar Kindly review the changes, do I need to add something else? :)